src/server/list_family.cc

```diff
@@ -317,16 +315,20 @@ OpResult<uint32_t> OpPush(const OpArgs& op_args, std::string_view key, ListDir d
   EngineShard* es = op_args.shard;
   PrimeIterator it;
   bool new_key = false;
+  DbSlice::AutoUpdater post_updater;
```

Comment:
you did not remove the explicit calls to pre and post update on this function

Reply:
good catch!